pandas/tests/groupby/test_transform.py

```diff
@@ -834,3 +834,13 @@ def demean_rename(x):
     tm.assert_frame_equal(result, expected)
     result_single = df.groupby('group').value.transform(demean_rename)
     tm.assert_series_equal(result_single, expected['value'])
+
+
+def test_groupby_transform_timezone_column():
```

Comment:
Not sure if it's too much here but might be worth creating a fixture for transformation functions and parametrizing based off of that here. That way we could ensure that the tz info doesn't get lost across all instead of just max.

Reply:
Fair point. For this test though, we need reducing functions that are applicable for timestamps though. All that make sense to me is `min` and `max` unless I am forgetting any other applicable functions. 

If the list is short, may be worth just parameterizing over directly instead?

Reply:
Yea agreed with your approach. I think it's worth while to start fixturizing these tests to ensure full coverage on particular subsets of algorithms, just don't know what those subsets should all be just yet. I think the way you've done it makes that refactor more apparent in the future if we ever get there